duoapi.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const (
+	version           = "1.0.0"
```

Comment:
I'd hesitate suggesting that this is major version 1.0.0. Go has [very specific](https://research.swtch.com/vgo-principles#compatibility) (and understandable) expectations when it comes to major version backwards compatibility. This means that breaking changes would need to be published in a new module, which [can be tricky.](https://blog.golang.org/v2-go-modules) 

To that end, we still have a fair number of API endpoints to build out and it feels like we're still getting a handle on how we want the structure of this library to look, so I wouldn't be surprised if at some point we have a breaking change we need to introduce.

All this to say, I'd recommend we start with something like `0.1.0` for now, since it's easier to go forward to `1.0.0` when we're comfortable than it is to go backwards 😄